### ISSUE TITLE
Point styling not applied to GLB point clouds

### ISSUE THREAD

**Issue by javagl** [CesiumGS/cesium#11520]
This _might_ be related to (or even be a duplicate of) https://github.com/CesiumGS/cesium/issues/11492 

It has originally been brought up in https://github.com/CesiumGS/3d-tiles-tools/issues/67 , but I suspect that this is in fact only a rendering issue in CesiumJS.

The following archive contains two tilesets:

- one with a PNTS file with classification info in the batch table
- one with a GLB file that contains the same information, converted into the `EXT_structural_metadata` information

It also contains a sandcastle for quick tests. The sandcastle defines a style for the tileset, and it shows this behavior:

![Cesium ToolsIssue67](https://github.com/CesiumGS/cesium/assets/5597569/55172746-d3fa-4750-80bb-2fddab51d51d)

For the case of the PNTS file, the style (green points) is applied correctly.
For the case of the GLB, the style is not applied. All the points are black. (Only after hovering the content, with the 3D Tiles Inspector enabled, the style is applied).
